src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-api-gateway-v1-monitoring-integration.mdx

```diff
@@ -1,21 +1,25 @@
 ---
-title: Amazon API Gateway monitoring integration
+title: Amazon API Gateway V1 monitoring integration
 tags:
   - Integrations
   - Amazon integrations
   - AWS integrations list
 metaDescription: "New Relic's AWS API Gateway monitoring integration: what data it reports, and how to enable it."
 redirects:
-  - /docs/integrations/amazon-integrations/aws-integrations-list/aws-api-gateway-monitoring-integration
+  - /docs/integrations/amazon-integrations/aws-integrations-list/aws-api-gateway-v1-monitoring-integration
   - /docs/aws-apigateway-integration
-  - /docs/infrastructure/amazon-integrations/amazon-integrations/aws-api-gateway-monitoring-integration
+  - /docs/infrastructure/amazon-integrations/amazon-integrations/aws-api-gateway-v1-monitoring-integration
 freshnessValidatedDate: never
 ---
 
 <Callout variant="important">
   Enable the [AWS CloudWatch Metric Streams integration](/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-metric-stream/) to monitor all CloudWatch metrics from your AWS services, including custom namespaces. Individual integrations are no longer our recommended option.
 </Callout>
 
+<Callout variant="tip">
```

Comment:
@rahul188 could you move this callout below the introductory paragraph? This will reduce reader fatigue from stacked callouts and should still catch users before they get too far.

Reply:
@rahul188 I moved the callouts around today. This separates the two callouts by the introductory paragraph.